bridge/bridge.go

```diff
@@ -119,8 +119,32 @@ func (b *Bridge) Sync(quiet bool) {
 	// Clean up services that were registered previously, but aren't
 	// acknowledged within registrator
 	if b.config.Cleanup {
-		log.Println("Cleaning up dangling services")
+		// Remove services if its corresponding container is not running
+		log.Println("Listing non-exited containers")
+		filters := map[string][]string{"status": {"created", "restarting", "running", "paused"}}
+		nonExitedContainers, err := b.docker.ListContainers(dockerapi.ListContainersOptions{Filters: filters})
+		if err != nil && quiet {
+			log.Println("error listing nonExitedContainers, skipping sync")
+			return
+		} else if err != nil && !quiet {
+			log.Fatal(err)
```

Comment:
Hmm, I'm not sure this should use Fatal, since that will exit the program immediately. There is another use of Fatal earlier in Sync, but that maybe should also just be logged as an error instead of exiting. @progrium ?